### PR TITLE
[UT] fix memory tracker ut

### LINKED ISSUE #5500: [UT] memory tracker UT couldn't work
**Describe the bug**
Memory Tracker UT coundn't work

### PR DESCRIPTION
**Pull request by stdpain**
## Proposed changes
will close #5500 

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

### CODE REVIEW

**Review by yangzhg — DISMISSED** [review]
LGTM

File: be/test/runtime/mem_limit_test.cpp [hunk 1]
```diff
@@ -23,89 +23,43 @@
...
-    t.Consume(10);
-    EXPECT_EQ(t.consumption(), 20);
-    t.Release(15);
-    EXPECT_EQ(t.consumption(), 5);
-    EXPECT_FALSE(t.LimitExceeded(MemLimit::HARD));
-}
+class MemTrackerTest : public testing::Test {
+public:
+    MemTrackerTest() {}
+};
```
Comment (acelyc111) [inline be/test/runtime/mem_limit_test.cpp:29]: Coz you are using `TEST`  only, so this class is not necessary?

**Review by EmmyMiao87 — DISMISSED** [review]
LGTM

File: be/test/runtime/mem_limit_test.cpp (hunk 1 above)
Comment (stdpain) [reply be/test/runtime/mem_limit_test.cpp:29]: Yes, I will remove it

**Review by EmmyMiao87 — APPROVED** [review]
LGTM

**Pull request merged**